README.md

```diff
@@ -44,12 +44,12 @@ import (
 )
 
 const (
-	db   = "protein"
 	tool = "entrez.example"
 )
 
 var (
 	clQuery = flag.String("query", "", "query specifies the search query for record retrieval (required).")
+	db      = flag.String("db", "protein", "db specifies the database to search")
```

Comment:
Probably you should check `*db` is not `""` at line 70 and add a "(required)" here.

Reply:
Doesn't the default take care of that?

Reply:
If you don't use the option, you get protein by default.

Reply:
Sorry, you're correct.